.travis.yml

```diff
@@ -1,19 +1,17 @@
+sudo: required
+
 language: ruby
-bundler_args: --without no_ci
-script:
-  - "bundle exec rspec"
-os:
-  - linux
-  - osx
-rvm:
-  - 2.1.0
-  - 2.2.0
-  - 2.3.0
```

Comment:
why remove the run of the unit test suite from TravisCI? In the docker setup we don't test against other Ruby versions